oauth2_client/providers/oauth2/urls.py

```diff
@@ -0,0 +1,48 @@
+import importlib
+import logging
+
+import six
+from django.conf.urls import include, url
+from importlib_metadata import ModuleNotFoundError
+
+from oauth2_client.providers.oauth2.provider import OAuth2Provider
+
+logger = logging.getLogger(__name__)
+
+
+def import_attribute(path):
+    """
+    Used for dynamic view imports for URL registration.
+    """
+    assert isinstance(path, six.string_types)
+    pkg, attr = path.rsplit('.', 1)
+    ret = getattr(importlib.import_module(pkg), attr)
+    return ret
+
+
+def default_urlpatterns(provider):
+    """
+    If Provider supports oauth-related views (_callback for now), register them
+    in a unified naming convention.
+    For example, OAuth2Provider's test callback view will be accessible at: /oauth2/login/callback/
```

Comment:
is it needed on prod environment?

Reply:
removed in the simplified architecture, marking this as resolved